.pipelines/templates/release-image-redactor.yml

```diff
@@ -0,0 +1,54 @@
+# Presidio-Image-Redactor Release: pypi and to ACR twice (with verison, latest) with 'public' tag for MCR release.
+
+trigger: none
+pr: none
+
+variables:
+    - group: Presidio-V2-CI
+stages:
+    - stage: Release
+      displayName: Release
+      jobs:
+          - job: version
+            displayName: 'Get the version number'
+            steps:
+                - bash: |
+                      set -eux  # exit on error
+                      ver=$(cat VERSION-IMAGE-REDACTOR)

```

Comment:
Is this the only change between this service and others? Maybe we can have it as a parameter and make it more generic.

Reply:
eventually I reverted to a single pipeline with a seperate job for the image-redactor